TodoListPosta/app/components/checkbox/styles.js

```diff
@@ -0,0 +1,17 @@
+import { StyleSheet } from 'react-native'
+
+const styles = StyleSheet.create({
+  box: {
+    height: 20,
+    width: 20,
+    borderWidth: 2,
+    borderColor: 'black',
+  },
+  inner: {
+    flex: 1,
+    margin: 2,
+    backgroundColor: 'rgba(0,0,0,0.8)',
```

Comment:
Usually, we extract colors as `constants` within a `src/constants/colors.js` file